src/app/courses/course-editor/course-editor.component.ts

```diff
@@ -34,60 +42,55 @@ export class CourseEditorComponent implements OnInit, CanComponentDeactivate {
           this.newCourse = { ...course };
         }
     });
+    this.courseForm = this.createForm();
+
+    this.store.dispatch(new LoadAuthors());
+
+    this.authors$ = this.store.pipe(
+      select(selectAuthors)
+    );
   }
 
-  public get authors(): string {
-    return this.newCourse.authors
-      .map(({firstName, lastName}) => lastName ?
-        `${firstName} ${lastName}` :
-        firstName)
-      .join(', ');
+  public createForm(): FormGroup {
+    return this.formBuilder.group({
+      title: [this.newCourse.title, [Validators.required, Validators.maxLength(50)]],
+      description: [this.newCourse.description, [Validators.required, Validators.maxLength(500)]],
```

Comment:
I think you add public property for integrate value from existing course.
Be better to avoid this one. By default we need to use this property for default parameter.
[{value: '', disabled: true}]
``description: ['', Validators.required, Validators.maxLength(500)],``
For set value form backend into form you could use `patchValue` instead